src/components/Sum.jsx

```diff
@@ -1 +1,15 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
+  <p>
+    Sum of
+    {` ${a} `}
+    and
+    {` ${b} `}
+    is
+    {` ${a + b}`}
+  </p>
```

Comment:
```suggestion
  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```

Use [template string ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)